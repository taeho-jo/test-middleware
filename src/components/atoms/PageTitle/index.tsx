import React from 'react';
// Components
import FlexBox from '../FlexBox';
// Styles
import { css } from '@emotion/react';
import { colors, commonStyles } from '../../../styles/Common.styles';
// Types
interface PropsType {
  title: string;
}

const PageTitle = ({ title }: PropsType) => {
  return (
    <FlexBox justify={'flex-start'} padding={'10px 0'}>
      <h1 css={mainTitleStyle}>{title}</h1>
    </FlexBox>
  );
};

export default PageTitle;

const mainTitleStyle = css`
  font-size: 30px;
  font-weight: 900;
  color: ${colors.green._500};
`;
