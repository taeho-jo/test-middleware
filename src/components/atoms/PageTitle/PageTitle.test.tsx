import React from 'react';
import { render } from '@testing-library/react';
import PageTitle from './index';

describe('PageTitle', () => {
  it('page title', () => {
    const utils = render(<PageTitle title={'page title'} />);
    expect(utils.container).toMatchSnapshot();
  });
});
