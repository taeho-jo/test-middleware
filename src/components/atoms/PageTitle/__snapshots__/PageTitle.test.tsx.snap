// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageTitle page title 1`] = `
<div>
  <div
    class="css-2it4no-flexBox"
  >
    <h1
      class="css-1bwimtt-mainTitleStyle"
    >
      page title
    </h1>
  </div>
</div>
`;
